cmd/osm-controller/osm-controller.go

```diff
@@ -296,3 +298,16 @@ func saveOrUpdateSecretToKubernetes(kubeClient clientset.Interface, ca certifica
 
 	return nil
 }
+
+func getCertificateManager(kubeClient kubernetes.Interface, kubeConfig *rest.Config) (certificate.Manager, debugger.CertificateManagerDebugger, error) {
+	switch *osmCertificateManagerKind {
+	case tresorKind:
+		return getTresorOSMCertificateManager(kubeClient, enableDebugServer)
+	case vaultKind:
+		return getHashiVaultOSMCertificateManager(enableDebugServer)
+	case certmanagerKind:
+		return getCertManagerOSMCertificateManager(kubeClient, kubeConfig, enableDebugServer)
+	default:
+		return nil, nil, fmt.Errorf("Unsupported Certificate Manager %s", *osmCertificateManagerKind)
```

Comment:
It would be nice to also state what the possible options are.

Reply:
At the time when CLI params are evaluated, the error message does print the supported kinds. Happy to add that here as well though.

Reply:
We might have use cases in the future where the charts are directly used, would be great to log this in the controller as well. Follow-up is ok.

Reply:
sounds good!